### DISCUSSION TITLE
Is it possible to overwrite a package?

### DISCUSSION THREAD

**Discussion by joewbrown** [AcademySoftwareFoundation/rez#1137] [Q&A]
To make a workflow that suits a specific part of our pipeline, we want to overwrite a package with each rez release, keeping the version number the same.

We were convinced we'd tested this and had it work, but now we can't get past `WARNING  Skipping 1/1: destination variant already exists`.

Were we mistaken, and this was never possible? 
If so, can you help us figure out an alternative workflow?

We have one particular package which we'd like a set of users to be able to commit to and release without as little overhead as possible. The package is part of a context that we rebuild when the other packages within it are released. , but we want this package to be free to be updated without a context rebuild, but such that the context still provides the latest code.

Our thinking was that if we just maintained a v0.1.0, we could overwrite that package with each release (we'd still have a history in git and we never need more than one version of this package deployed at any given time).

**Comment by davidlatwe** [comment]
Hi @joewbrown , by "context rebuild", did you mean regenerate the context `.rxt` file with same set of package requests ?

**Reply by joewbrown** [reply]
Yes exactly, @davidlatwe  :)

**Reply by davidlatwe** [reply]
I think overriding same version of package in production could be dangerous sometimes, so better to avoid that if possible.

I am not sure if I tried this before, but maybe you could try `rez-env --patch` ? The same RXT should be reusable with new package patching.
